__tests__/app.test.js

```diff
@@ -0,0 +1,43 @@
+const app = require("../app-files/app");
+const seed = require("../db/seeds/seed");
+const request = require('supertest')
+const db = require("../db/connection");
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed({ articleData, commentData, topicData, userData });
+});
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api/topics", () => {
+  test("GET: 200 responds with an array of topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        const {topics} = response.body;
+        expect(topics).toHaveLength(3);
```

Comment:
Great that you're checking the length of the array to avoid false positives if passed an empty array 👍